webapp/src/app/shared/phaser/helper.ts

```diff
@@ -102,10 +100,12 @@ export class Helper {
 		}
 		
 		return new Promise(res => {
-			scene.load.image(key, Globals.URL + key);
-			scene.load.once('complete', () => res(true));
-			scene.load.once('loaderror', () => res(false));
-			scene.load.start();
+			Globals.modloaderService.getResourcePath(key).subscribe((truePath) => {
```

Comment:
Use `await` for clean code

Reply:
There would no longer be a point to getJsonPromise. I don't even see the reason it's still there.

Reply:
I was looking at the wrong code.